source/_integrations/coinbase.markdown

```diff
@@ -9,46 +9,35 @@ ha_iot_class: Cloud Polling
 ha_domain: coinbase
 ---
 
-The `coinbase` integration lets you access account balances and exchange rates from [coinbase](https://coinbase.com).
+The `coinbase` integration lets you access account balances and exchange rates from [Coinbase](https://coinbase.com).
 
-You will need to obtain an API key from coinbase's [developer site](https://www.coinbase.com/settings/api) to use this component. You need to give read access to `wallet:accounts` in order for the integration to access relevant data.
+You will need to obtain an API key from the API section in Coinbase's [User Settings](https://www.coinbase.com/settings/api) to use this integration. You need to select the account wallet or wallets (e.g. "BTC Wallet") that you wish to show in Home Assistant and give read access to `wallet:accounts` in order for the integration to access relevant data. It is worth noting that once you close the New API Key popup on Coinbase you will not be able to see the API Secret again.
 
 ## Configuration
 
-To set it up, add the following information to your `configuration.yaml` file:
+The Coinbase integration is configured through Home Assistant's user interface, go the **Configuration** page then the **Integrations** page, click the `+` button and select Coinbase from the list of integrations. There will be a short delay while the integration's dependencies are installed after which you will be presented with a form to enter the required information to configure the integration.
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```

And remove the header above and below this text (`## configuration` & `### Configuration Values`)